script.tablo/addon.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.tablo"
+       name="Tablo"
+       version="1.0.0"
+       provider-name="Nuvyyo">
+    <requires>
+        <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
We have been enforcing this for new add-ons. It's easier all around. We don't have to move it to a new repo when support for the old one stops, people should be updating to newer versions and we encourage add-on authors to take advantage of the newer